### ISSUE TITLE
InvalidOperationException in 1.5.3.4

### ISSUE THREAD

**Issue by hduregger** [ciribob/DCS-SimpleRadioStandalone#265]
* Silently crashes SRS
* SRS 1.5.3.4
* Connected to Bluflag on Buddyspike 5.189.162.17:5014
* F-18 radios tuned and I can hear others talk
* After takeoff and some flight time SRS crashes in the background.
* Can't remember ever seeing SRS crash before.
* I've used the older version with the F-18 bug with the F-18 before and it didn't crash I think (did not work though of course). Not sure if it didn't crash because it did not work and maybe I didn't realize it crashed while I was flying.
* Probably unrelated: I can't manually connect to the Buddyspike SRS server mentioned above. I have to use the autoconnecting feature where it joins as soon as I join the DCS server.

Stacktrace from Windows 10 Event Logs

```
System.InvalidOperationException
   bei System.ThrowHelper.ThrowInvalidOperationException(System.ExceptionResource)
   bei System.Collections.Generic.List`1+Enumerator[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].MoveNextRare()
   bei SharpConfig.Section.FindSetting(System.String)
   bei SharpConfig.Section.Contains(System.String)
   bei Ciribob.DCS.SimpleRadio.Standalone.Client.Settings.SyncedServerSettings.GetSetting(Ciribob.DCS.SimpleRadio.Standalone.Common.Setting.ServerSettingsKeys)
   bei Ciribob.DCS.SimpleRadio.Standalone.Client.UI.MainWindow.RedrawUITick(System.Object, System.EventArgs)
   bei System.Windows.Threading.DispatcherTimer.FireTick(System.Object)
   bei System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   bei System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   bei System.Windows.Threading.DispatcherOperation.InvokeImpl()
   bei MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(System.Object)
   bei System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   bei System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   bei System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   bei MS.Internal.CulturePreservingExecutionContext.Run(MS.Internal.CulturePreservingExecutionContext, System.Threading.ContextCallback, System.Object)
   bei System.Windows.Threading.DispatcherOperation.Invoke()
   bei System.Windows.Threading.Dispatcher.ProcessQueue()
   bei System.Windows.Threading.Dispatcher.WndProcHook(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   bei MS.Win32.HwndWrapper.WndProc(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   bei MS.Win32.HwndSubclass.DispatcherCallbackOperation(System.Object)
   bei System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   bei System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   bei System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority, System.TimeSpan, System.Delegate, System.Object, Int32)
   bei MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)
   bei MS.Win32.UnsafeNativeMethods.DispatchMessage(System.Windows.Interop.MSG ByRef)
   bei System.Windows.Threading.Dispatcher.PushFrameImpl(System.Windows.Threading.DispatcherFrame)
   bei System.Windows.Application.RunDispatcher(System.Object)
   bei System.Windows.Application.RunInternal(System.Windows.Window)
   bei DCS_SR_Client.App.Main()
```

**Comment by MorpheusXAUT** [comment]
Hi @hduregger,

thanks for reporting this issue, sorry about the crash - we're aware of it and a fix has already been pushed to our `beta` branch.
We'll hopefully have an update out within the next few days to address this bug.

In the meantime, you could keep using the older version (`1.5.3.2`) and replace the `DCS-SimpleRadioStandalone.lua` as described at the very top of the [`1.5.3.2` release notes](https://github.com/ciribob/DCS-SimpleRadioStandalone/releases/tag/1.5.3.2).

Regarding your issue with the BlueFlag SRS server: it's indeed unrelated to this - the SRS server was moved and is now available at `54.36.126.186:5014`. You should be fine if you still just stick to auto-connect though, that'll have the updated IP.

**Comment by hduregger** [comment]
Hey @MorpheusXAUT,

thank you for the quick reply and the info about blueflag! Will try the older SRS version with the patch.

**Comment by ciribob** [comment]
Fixed in Release 1.5.3.5
